recipes/virema/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: virema
+  version: "0.6"
+
+source:
+  fn: ViReMa_0.6.zip
+  url: https://sourceforge.net/projects/virema/files/ViReMa_0.6/ViReMa_0.6.zip
+  md5: 2e15fcea133a64b4b043dba39d87f289
+  patches:
+    - path.patch
```

Comment:
What is this patch doing?

Reply:
@bgruening , the scripts are python scripts, so this patch adds "/opt/anaconda1anaconda2anaconda3/bin/python", and there is also "^M" in the python script after this (:?), so I remove that as well. Is that ok ?

Reply:
You should add `#!/usr/bin/env python` and not the long /opt/anaconda prefix. I would not touch the line endings if not needed.

Reply:
Thanks for the suggestion @bgruening , I fixed that in the newest commit.

Reply:
Can you try to make the patch smaller and not change line endings. Its hard to read the patch this way.

Reply:
Sorry about that @bgruening , it is fixed now.